src/components/sidebar/sidebar.jsx

```diff
@@ -11,6 +11,16 @@ const SidebarBlock = styled.aside`
     transform: ${props =>
         props.isVisible ? 'translate3d(0px, 0px, 0px)' : 'translate3d(-100%, 0px, 0px)'};
     will-change: transform;
+
+    @media screen and (max-width: 375px) {
+        position: fixed;
+        top: 0;
+        bottom: 0;
+        left: 0;
+        right: 0;
+        z-index: 1;
```

Comment:
can we put it in variables, so we know how these values work and overlap?

Reply:
Added device media variables. Changed layout css approach. Solved.